src/views/editor/index.vue

```diff
@@ -416,7 +429,7 @@ export default {
        */
       return this
       .compiled
```

Comment:
Instead of `this.compiled` you can use `this.instance`

Reply:
@nduchak yep! I'm going over the docs